values.yaml

```diff
@@ -15,7 +15,7 @@ docker:
 ##
 cog:
   name: cog
-  serviceType: ClusterIP
+  serviceType: NodePort
```

Comment:
I'm using NodePort on my setup and it works. I'm not sure of the value of ClusterIP vs NodePort.